frontend/src/bundles/common/components/chip/chip.tsx

```diff
@@ -0,0 +1,23 @@
+import { Close } from '@mui/icons-material';
+import { Chip as MUIChip } from '@mui/material';
+import clsx from 'clsx';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    label: string;
+    onDelete?: () => void;
+};
+
+const Chip = ({ label, onDelete }: Properties): JSX.Element => {
+    return (
+        <MUIChip
+            className={clsx(styles.chip, { [styles['delete-chip']]: onDelete })}
```

Comment:
Use please _getValidClassNames_ helpers

```suggestion
            className={getValidClassNames(styles.chip, { [styles['delete-chip']]: onDelete })}
```

Reply:
thanks, I've changed it